patterns/homepage.php

```diff
@@ -0,0 +1,230 @@
+<p class="has-text-align-center has-league-gothic-font-family has-primary-color has-text-color has-3-2-x-large-font-size" style="line-height:1;text-transform:uppercase">grow your writing Skills with Course</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:paragraph {"align":"center","style":{"typography":{"letterSpacing":"-1%"}},"textColor":"primary","className":"has-eb-garamond-font-family","fontSize":"small"} -->
+<p class="has-text-align-center has-eb-garamond-font-family has-primary-color has-text-color has-small-font-size" style="letter-spacing:-1%">Writing is powerful! Allows you to articulate and explain yourself to others. When done well, it allows you to tell stories that could fascinate millions.</p>
+<!-- /wp:paragraph -->
+
+<!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
+<div class="wp-block-buttons"><!-- wp:button {"backgroundColor":"primary","style":{"border":{"radius":"8px"}},"fontSize":"x-small","fontFamily":"league-gothic"} -->
+<div class="wp-block-button has-custom-font-size has-league-gothic-font-family has-x-small-font-size"><a class="wp-block-button__link has-primary-background-color has-background" style="border-radius:8px">START LEARNING NOW</a></div>
+<!-- /wp:button --></div>
+<!-- /wp:buttons -->
+
+<!-- wp:spacer {"height":"10px"} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer --></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"align":"full","style":{"spacing":{"padding":{"top":"20px","bottom":"20px"}}},"backgroundColor":"primary","textColor":"foreground","className":"course-home-page-responsive-padding","layout":{"type":"default","contentSize":"1200px"}} -->
+<div class="wp-block-group alignfull course-home-page-responsive-padding has-foreground-color has-primary-background-color has-text-color has-background" style="padding-top:20px;padding-bottom:20px"><!-- wp:spacer {"height":"10px"} -->
+<div style="height:10px" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:group {"backgroundColor":"primary","textColor":"tertiary","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"space-between","verticalAlignment":"center"}} -->
+<div class="wp-block-group has-tertiary-color has-primary-background-color has-text-color has-background"><!-- wp:group {"style":{"spacing":{"padding":{"left":"20px"}}},"borderColor":"tertiary","className":"is-style-group-left-border","layout":{"type":"flex","flexWrap":"nowrap","justifyContent":"left"}} -->
+<div class="wp-block-group is-style-group-left-border has-border-color has-tertiary-border-color" style="padding-left:20px"><!-- wp:heading {"level":3,"style":{"typography":{"textTransform":"uppercase"}},"className":"has-league-gothic-font-family"} -->
+<h3 class="has-league-gothic-font-family" style="text-transform:uppercase">COURSE LIST</h3>
```

Comment:
This string should be translatable

Reply:
Translated here https://github.com/Automattic/sensei-theme/pull/44/commits/efb98493e7afa7a1576dbb5b291a0b5cd2a12328